.github/ISSUE_TEMPLATE/flaky-test.yml

```diff
@@ -15,15 +15,15 @@ body:
     id: link
     attributes:
       label: "Link"
-      description: "Link to file/line number in GitHub."
+      description: "Link to failing Github Action"
```

Comment:
Github action logs aren't kept forever, therefore linking only the gh action isn't useful for future reference. Better have both, action and file/number